app/controllers/rides_controller.rb

```diff
@@ -17,7 +17,9 @@ def create
     @ride = current_user.rides.build(ride_params)
 
     if @ride.save
-      redirect_to ride_path(@ride.id)
+      @ride.add_driver current_user
+      flash[:notice] = t 'ride.save.success_html', url: ride_path(@ride)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.